src/test/java/sigma/TaskListTest.java

```diff
@@ -0,0 +1,74 @@
+package sigma;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import sigma.Parser;
+import sigma.TaskList;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import static org.junit.jupiter.api.Assertions.*;
+import sigma.task.Task;
+public class TaskListTest {
+    TaskList taskList;
+    Parser parser;
```

Comment:
I think here can explicitly declare the access modifiers of the class variables.